src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1373,11 +1390,6 @@ public boolean isSiegeImmunityActive() {
 		}
 	}
 
-	public int getRevoltCooldownRemainingMinutes() {
```

Comment:
Was this supposed to get nuked in this commit?

Reply:
Yes.
As I was merging in the latest master,
I noticed this completely unused method and removed it.